src/Symfony/Component/VarDumper/Cloner/Stub.php

```diff
@@ -39,7 +39,7 @@ class Stub
     public $position = 0;
     public $attr = [];
 
-    private static $defaultProperties = [];
+    protected static $defaultProperties = [];
```

Comment:
I changed the visibility because this line catch an Exception : https://github.com/symfony/symfony/pull/33114/files#diff-2e2bdf9c982ba28c381992923d760aceL60